test/controllers/projects/samples/transfer_controller_test.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+module Projects
+  module Samples
+    class TransferControllerTest < ActionDispatch::IntegrationTest
+      setup do
+        sign_in users(:john_doe)
+        @sample1 = samples(:sample1)
+        @sample2 = samples(:sample2)
+        @project1 = projects(:project1)
+        @project2 = projects(:project2)
+        @namespace = groups(:group_one)
+      end
+
+      test 'should get new if owner' do
+        get new_namespace_project_transfer_path(@namespace, @project1)
+        assert_response :success
+      end
+
```

Comment:
Please also add a test for if a non-owner tries to visit the `new ` page, and also one for if a non-member of the project or one of it's ancestor groups tries to initiate a transfer

Reply:
Please see 618ad723f47b3c80162717460110a60c2afbe49f.

Reply:
Thanks!